src/input_arguments_handling.jl

```diff
@@ -0,0 +1,122 @@
+# These are the user-provided arguments to `parse_file` or `setup_parser`.
```

Comment:
these are only _some_ of the user-provided inputs, no? what about `delim` etc?

Reply:
Right, these are the ones that don't belong to `Parsers.Options`